### PR TITLE
support to change rtt dynamically in gym

### PR DESCRIPTION
**Pull request by Azson**

### CODE REVIEW

File: ns-app/scratch/webrtc_test/trace_player.cc [hunk 1]
```diff
@@ -40,6 +42,11 @@ void TracePlayer::LoadTrace() {
         TraceItem ti;
         ti.capacity_ = lexical_cast<decltype(ti.capacity_)>(trace["capacity"]);
         ti.duration_ms_ = lexical_cast<decltype(ti.duration_ms_)>(trace["duration"]);
+        if (trace.find("rtt") != trace.end()) {
+            ti.rtt_ms_ = lexical_cast<decltype(ti.rtt_ms_)>(trace["rtt"]);
+        } else {
+            ti.rtt_ms_ = 0;
```
Comment (Pterosaur) [inline ns-app/scratch/webrtc_test/trace_player.cc]: Don't set 0 as the default value, try boost::optional.

File: ns-app/scratch/webrtc_test/trace_player.cc [hunk 2]
```diff
@@ -55,6 +62,8 @@ void TracePlayer::PlayTrace(size_t trace_index) {
     const auto &trace = traces_[trace_index];
     for (size_t i = 0; i < nodes_.GetN(); i++) {
         auto node = nodes_.Get(i);
+        // set delay in channel
+        node->GetDevice(0)->GetChannel()->SetAttribute("Delay", StringValue(std::to_string(trace.rtt_ms_/2.0) + "ms"));
```
Comment (Pterosaur) [inline ns-app/scratch/webrtc_test/trace_player.cc]: Please check whether RTT was assigned.

**Comment by Pterosaur** [comment]
Please correct the test

File: ns-app/scratch/webrtc_test/trace_player.cc (hunk 1 above)
Comment (Azson) [reply ns-app/scratch/webrtc_test/trace_player.cc]: fixed in commit 03f42d6

File: ns-app/scratch/webrtc_test/trace_player.cc (hunk 2 above)
Comment (Azson) [reply ns-app/scratch/webrtc_test/trace_player.cc]: fixed in commit 03f42d6

**Review by Pterosaur — APPROVED** [review]

**Pull request merged**